freshmaker/handlers/koji/rebuild_flatpak_application_on_module_ready.py

```diff
@@ -134,11 +134,9 @@ def _handle_or_skip(self, event):
         images_nvr = ",".join([rebuild_image.nvr for rebuild_image in rebuild_images])
         self.log_info("Following images %s will be rebuilt" % images_nvr)
 
-        self._record_builds(rebuild_images, image_modules_mapping)
+        builds = self._record_builds(rebuild_images, image_modules_mapping)
 
-        # TODO: Return empty list so far as this method is still in progress.
-        # Will think about whether to return the real BaseEvent objects in future.
-        return []
+        self.start_to_build_images(builds.values())
```

Comment:
According to docstring of `BaseHandler.handle()` this needs to return list.

Reply:
Ack. Fixed